### PR TITLE
Show header with the widget name above the preview area.

### PR DESCRIPTION
**Pull request by scheglov**
It seems that users are confused otherwise, and selection in the tree is not enough, so we decided to spend some vertical space for explanation. OTOH, this header might eventually host some additional action buttons (which don't fit / not appropriate for the main view toolbar?).

image

### CODE REVIEW

**Comment by devoncarew** [comment]
Should we use the regular toolbar class here to show the name? It would likely have similar padding and look as other areas in IntelliJ.

I think we should also just use the name as the label (`MyWidget`) instead of the longer `UI preview of MyWidget's build method:`.

File: src/io/flutter/preview/PreviewArea.java [hunk 1]
```diff
@@ -152,10 +159,18 @@ public void show(@NotNull FlutterOutline unitOutline, @NotNull JsonObject render
       return;
     }
 
+    final Element widgetClassElement = widgetOutline.getDartElement();
+    if (widgetClassElement != null) {
+      header.setText("UI preview of " + widgetClassElement.getName() + "’s build() method:");
```
Comment (pq) [inline src/io/flutter/preview/PreviewArea.java]: Perhaps something a little more terse?  What about just:

    widgetClassElement.getName() + ".build():"

yielding something like:

 **MyWidget.build():**

Comment (scheglov) [reply src/io/flutter/preview/PreviewArea.java]: PTAL

Comment (scheglov) [reply src/io/flutter/preview/PreviewArea.java]: image

**Pull request merged**